examples/i18n/src/i18n.rs

```diff
@@ -0,0 +1,96 @@
+use fluent::{FluentArgs, FluentBundle, FluentResource};
+use strum_macros::EnumIter;
+use unic_langid::LanguageIdentifier;
+
+use std::borrow::Borrow;
+use std::str::FromStr;
+
+use crate::resource::Resource;
```

Comment:
I think you can replace `crate` with `super` - your choice

Reply:
Super. Done.